  Building modules, stage 2.
  MODPOST 759 modules
ERROR: "__divdi3" [drivers/crypto/hifn_795x.ko] undefined!

--- a/drivers/crypto/Kconfig	2008-02-26 09:14:14.000000000 +0000
+++ b/drivers/crypto/Kconfig	2008-02-26 09:15:11.000000000 +0000
@@ -149,6 +149,7 @@
 	select CRYPTO_BLKCIPHER
 	select HW_RANDOM if CRYPTO_DEV_HIFN_795X_RNG
 	depends on PCI
+	depends on (!MIPS && !ARM) || BROKEN
 	help
 	  This option allows you to have support for HIFN 795x crypto adapters.
 
